### ISSUE TITLE
Mobile device specific (notification-bar-like) placement (feature request)

### ISSUE THREAD

**Issue by dennis97519** [stevenbenner/jquery-powertip#144]
Is it possible to make a mobile wiki style tooltip that pops up from the bottom? and trigger this style on mobile user agent string or a mobile flag.

**Comment by stevenbenner** [comment]
Not with PowerTip today. At least not easily.

You can try to detect if you're on a mobile device or in a small virtual screen resolution then disable the PowerTip tooltips entirely and use some custom code, or different plugin designed to do those kinds of popups. But that sounds kind of painful to me.

I haven't had to do this yet, so I can't recommend any good tools to do this. But I'm confident that someone (probably many someones) has written scripts to do exactly those types of popups.

But to rephrase this as a feature request; I could consider adding new tooltip positions based on the window edges instead of an element in the page. For example `{ placement: 'screenBottom' }`.

**Comment by dennis97519** [comment]
Yeah, the new tooltip positions would be good.
